themes-default/slim/views/vue-components/app-header.mako

```diff
@@ -0,0 +1,216 @@
+                    <span v-if="toolsBadgeCount > 0" :class="'badge' + toolsBadgeClass">{{ toolsBadgeCount }}</span>
+                    <b class="caret"></b>
+                    </app-link>
+                    <ul class="dropdown-menu">
+                        <li><app-link href="news/"><i class="menu-icon-news"></i>&nbsp;News <span v-if="config.news.unread > 0" class="badge">{{ config.news.unread }}</span></app-link></li>
+                        <li><app-link href="IRC/"><i class="menu-icon-irc"></i>&nbsp;IRC</app-link></li>
+                        <li><app-link href="changes/"><i class="menu-icon-changelog"></i>&nbsp;Changelog</app-link></li>
+                        <li><app-link :href="config.donationsUrl"><i class="menu-icon-support"></i>&nbsp;Support Medusa</app-link></li>
+                        <li role="separator" class="divider"></li>
+                        <li v-if="config.logs.numErrors > 0"><app-link href="errorlogs/"><i class="menu-icon-error"></i>&nbsp;View Errors <span class="badge btn-danger">{{config.logs.numErrors}}</span></app-link></li>
+                        <li v-if="config.logs.numWarnings > 0"><app-link :href="'errorlogs/?level=' + warningLevel"><i class="menu-icon-viewlog-errors"></i>&nbsp;View Warnings <span class="badge btn-warning">{{config.logs.numWarnings}}</span></app-link></li>
+                        <li><app-link href="errorlogs/viewlog/"><i class="menu-icon-viewlog"></i>&nbsp;View Log</app-link></li>
+                        <li role="separator" class="divider"></li>
+                        <li><app-link :href="'home/updateCheck?pid=' + config.pid"><i class="menu-icon-update"></i>&nbsp;Check For Updates</app-link></li>
+                        <li><app-link :href="'home/restart/?pid=' + config.pid" class="confirm restart"><i class="menu-icon-restart"></i>&nbsp;Restart</app-link></li>
+                        <li><app-link :href="'home/shutdown/?pid=' + config.pid" class="confirm shutdown"><i class="menu-icon-shutdown"></i>&nbsp;Shutdown</app-link></li>
+                        <li v-if="username"><app-link href="logout" class="confirm logout"><i class="menu-icon-shutdown"></i>&nbsp;Logout</app-link></li>
+                        <li role="separator" class="divider"></li>
+                        <li><app-link href="home/status/"><i class="menu-icon-info"></i>&nbsp;Server Status</app-link></li>
+                    </ul>
+                    <div style="clear:both;"></div>
+                </li>
+            </ul>
+        </div><!-- /.navbar-collapse -->
+    </div><!-- /.container-fluid -->
+</nav>
+<!-- END HEADER -->
+</script>
+<%!
+    import json
+    from medusa import app
+%>
+<script>
+Vue.component('app-header', {
+    template: '#app-header-template',
+    data() {
+        return {
+            // Python conversions
+            <% has_emby_api_key = json.dumps(app.EMBY_APIKEY != '') %>
+            hasEmbyApiKey: ${has_emby_api_key},
```

Comment:
@OmgImAlexis 
I think this the last blocker to finishing this component (and maybe moving to `.vue` file).
I can't add the API key to the endpoint (#4404), and we never decided on a solution for the passwords / API keys.

Reply:
We should allow `PATCH` for them for now. I'm really not okay with sending the API keys and passwords back to the client if we don't have to.

Reply:
Here I need to know if the API key is set, not patch it.
So what I'm saying is, there was a discussion on Slack about how to handle situations where you need to know if it's set (excluding the config pages where we currently have the value in the HTML).
That discussion never reached its conclusion - so we need to figure it out.

I've decided to just remove the emby API key check for now, just so I can move forward with this.
Left a comment in place though.

Reply:
Why is the value needed? We may be able to work around this in another way.